x-pack/plugin/src/test/resources/rest-api-spec/test/ml/jobs_crud.yml

```diff
@@ -516,70 +491,6 @@
             "groups": ["job-crud-update-group-name-clash"]
           }
 
----
-"Test cannot decrease model_memory_limit below current usage":
```

Comment:
This test is now covered in `org.elasticsearch.xpack.ml.integration.AnomalyJobCRUDIT`

Reply:
All the tests in this file that have been deleted are listed in the blacklist in `x-pack/plugin/ml/qa/ml-with-security/build.gradle` - please delete them from there.